components/css/newsletter-text-only/newsletter-text-only-1.0.0.scss

```diff
@@ -8,11 +8,11 @@
 @import '../global/top';
 @import '../global/nav';
 @import '../global/breadcrumb';
-@import 'contentPage';
 @import '../global/form';
 @import '../global/footer';
 @import '../global/bottom';
 @import '../global/bootstrap-fixes';
 @import '../mixins/calculations';
 @import '../mixins/fonts';
-@import '../partials/button';
\ No newline at end of file
+@import '../partials/button';
+@import 'contentPage';
```

Comment:
@jacobvenable we moved the page scss import here. Is that going to be okay. Previously it was in the default position from other content pages.

Reply:
Ya, that should be fine.